goguen/default.nix

```diff
@@ -0,0 +1,53 @@
+with builtins; with import ../lib.nix;
+{ pkgs ? import goguenNixpkgs {}
+, ...
+}:
+with pkgs.lib; with pkgs;
+let
+  listVersions = drvs: stdenv.mkDerivation {
+    name = "versions";
+    src = ./.;
```

Comment:
you probably want `runCommand` rather then `mkDerivation` here

Reply:
I'm looking towards removing the initial G-ala-C implementation at some point in the future.
We can't do it for a reason right now.
In this light, I'm not sure it'd be an economic expenditure of effort to focus on making this pretty.